modules/gdnative/include/pluginscript/godot_pluginscript.h

```diff
@@ -127,6 +127,7 @@ typedef struct {
 	const char **string_delimiters; // nullptr  terminated array
 	godot_bool has_named_classes;
 	godot_bool supports_builtin_mode;
+	godot_bool can_inherit_from_file;
```

Comment:
Is `godot_bool` value initialized automatically?

Reply:
nop, `godot_bool` is just and alias to a base type (defined as `typedef bool godot_bool;`)
In fact nothing is initialized automatically here given this .h is part of the gdnative C API (so this structure is created from C code, so everything must be explicitly initialized/teardown)